lib/zold/node/farm.rb

```diff
@@ -48,7 +48,7 @@ def initialize(invoice, cache, log: Log::Quiet.new)
     end
 
     def best
-      load
+      load.compact
```

Comment:
@emilianodellacasa by putting compact here, you force me to know that you can return `nil`s. I'd prefer not to care about that.